setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+  name = 'bplist',
+  packages = ['bplist'],  
+  version = '0.1',
+  description = 'Apple binary Property List reader/writer for Python',
+  install_requires=[],
+  author = 'Vladimir Pouzanov',
+  author_email = 'farcaller@gmail.com',
+  url = 'https://github.com/gamikun/bplist-python',
```

Comment:
That's because I published some months ago, but i decided to make the pull request today. I just fixed it.